Service/Extend.php

```diff
@@ -24,4 +24,4 @@ class Extend
     public const COLLECT_ALL_ORDERS = true;
 
     public const SHIPPING_PROTECTION_LABEL = 'Shipping Protection';
-}
\ No newline at end of file
+}
```

Comment:
dunno what this change is all about - i did use this file to test some linting stuff out which all looked good; my vscode is set up to ensure newlines at the end of every file by default on save, and it's showing up in my local, but not in this diff 🤷